### PR TITLE
Fix PRC charts displaying undefined

### PR DESCRIPTION
**Pull request by paulineribeyre**
Jira Ticket: COV-595

bug introduced in https://github.com/uc-cdis/data-portal/pull/773

### Bug Fixes
- Covid-19 dashboard: fix US charts not loading because of an undefined variable

### CODE REVIEW

File: src/Covid19Dashboard/WorldMapChart/index.jsx [hunk 1]
```diff
@@ -192,7 +192,8 @@ class WorldMapChart extends React.Component {
 
       // density map data contains fips and iso3;
       // choropleth map data contains FIPS and iso_a3.
-      const fips = `${feature.properties.FIPS || feature.properties.fips}`;
+      let fips = feature.properties.FIPS || feature.properties.fips;
+      fips = fips ? `${fips}` : fips; // convert to string if not undefined
```
Comment (ocshawn) [inline src/Covid19Dashboard/WorldMapChart/index.jsx:196]: this will evaluate to false if vale of fips is 0, is this acceptable?

Comment (ocshawn) [reply src/Covid19Dashboard/WorldMapChart/index.jsx:196]: can use 
`typeof fips !== 'undefined'`

Comment (paulineribeyre) [reply src/Covid19Dashboard/WorldMapChart/index.jsx:196]: fips 0 doesn't exist afaik but i'll update it to be safe

**Review by ocshawn — APPROVED** [review]
looks good

**Pull request merged**